projects/plugins/search/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1-beta] - 2022-07-12
+### Changed
+- E2E tests: bump @playwright/test and allure-playwright versions
```

Comment:
I think these are the Jetpack-the-plugin changelog entries rather than the ones for the Search package perhaps?

Reply:
yeah.. I was wondering that. It was auto-generated using `jetpack release plugins/search changelog -b` though 🤔